plugins/catalog-backend-module-openapi-spec/catalog-info.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-plugin-catalog-backend-module-openapi-spec
+  title: '@backstage/plugin-catalog-backend-module-openapi-spec'
+spec:
+  lifecycle: experimental
+  type: backstage-backend-plugin-module
+  owner: maintainers
```

should this rather be the OpenAPI Maintainers?

```suggestion
  owner: openapi-tooling-maintainers
```

---

Hmm, maybe? I think this package was created before `openapi-tooling-maintainers` was a thing. @Rugvip thoughts on updating this and the codeowners?